types/steam-user/index.d.ts

```diff
@@ -646,6 +646,14 @@ declare class SteamUser extends EventEmitter {
      */
     removeFriendFromGroup(groupID: any, usersteamID: SteamID | string, callback?: (err: Error | null) => void): Promise<void>;
 
+    /**
+     * Retrieves a list of friends that have played or used an app.
+     * @param appID - The ID of the app you want to check
+     * @param [callback]
+     * @since 4.20.0
+     */
+    getFriendsThatPlay(appID: number, callback?: (err: Error | null, response: { friends: SteamID[] }) => void): Promise<{ friends: SteamID[] }>;
```

Comment:
The docs don’t say this returns a Promise. You sure this is right? Often with async APIs that support callbacks _or_ Promises, the Promise is only returned if a callback is _not_ supplied, in which case you should model this with an overload. But I haven’t tried to find this in the source code yet myself.

Reply:
I did a quick test:
```js
steamuser.getPlayerCount(730, (_, playerCount) => {
    console.log('Callback: %d', playerCount);
}).then(({ playerCount }) => {
    console.log('Promise: %d', playerCount);
}).catch();
```
Promise and Callback bot returned a value.

DoctorMcKay has the necessary code in his [node-stdlib](https://github.com/DoctorMcKay/node-stdlib) to do that:
[timeoutCallbackPromise](https://github.com/DoctorMcKay/node-stdlib#timeoutcallbackpromisetimeout-callbackargs-callback-isoptional-executor)